.golangci.yml

```diff
@@ -149,16 +149,16 @@ linters:
   disable-all: true
   enable:
     ## enabled by default
-    - deadcode # Finds unused code
```

Comment:
Note sure it's okay to disable those

Reply:
I disabled those based on the WARN messages. They give out deprecation warnings and instructs the user to remove them. I can switch them back if you prefer.

Reply:
You are correct, however, `unused` was already enabled in the linter, so we don't need to duplicate it. Fixed in the master branch. 🙌🏻